FWCore/Concurrency/interface/SerialTaskQueueChain.h

```diff
@@ -68,98 +63,92 @@ namespace edm {
      * In that way the core is not idled while waiting.
      * \param[in] iAction Must be a functor that takes no arguments and return no values.
      */
-    template<typename T>
+    template <typename T>
     void pushAndWait(T&& iAction);
-    
+
     unsigned long outstandingTasks() const { return m_outstandingTasks; }
-    std::size_t numberOfQueues() const {return m_queues.size(); }
+    std::size_t numberOfQueues() const { return m_queues.size(); }
+
   private:
-    
     // ---------- member data --------------------------------
     std::vector<std::shared_ptr<SerialTaskQueue>> m_queues;
     std::atomic<unsigned long> m_outstandingTasks{0};
-    
-    template<typename T>
+
+    template <typename T>
     void passDownChain(unsigned int iIndex, T&& iAction);
-    
-    template<typename T>
-    void actionToRun(T&& iAction);
 
+    template <typename T>
+    void actionToRun(T&& iAction);
   };
-  
-  template<typename T>
+
+  template <typename T>
   void SerialTaskQueueChain::push(T&& iAction) {
     ++m_outstandingTasks;
-    if(m_queues.size() == 1) {
-      m_queues[0]->push( [this,iAction]() mutable {this->actionToRun(iAction);} );
+    if (m_queues.size() == 1) {
+      m_queues[0]->push([this, iAction]() mutable { this->actionToRun(iAction); });
     } else {
       assert(!m_queues.empty());
-      m_queues[0]->push([this, iAction]() mutable {
-        this->passDownChain(1, iAction);
-      });
+      m_queues[0]->push([this, iAction]() mutable { this->passDownChain(1, iAction); });
```

Comment:
I find this harder to read. Putting the body of a lambda on a separate line seems better to me.

Reply:
you and @Dr15Jones make the same comment. We can forbid short functions from being on the same line - however I believe this change came out of a previous iteration (to be confirmed)

Reply:
whops, I misread - so we already had this

AllowShortIfStatementsOnASingleLine: false
AllowShortLoopsOnASingleLine: false

but this is all one command, just a complex one. I'm looking into if there is a hook or not